neuronsimulator/utils.py

```diff
@@ -11,11 +11,78 @@
 
 class WormfunconnToPlot:
     """
-    contains a set of methods for calling wormfuconn package to generate parameters plot,
-    and url for displaying neural response plot(s) on the webpage
+    contains a set of methods for calling wormfuconn package to parse parameters and generate
+    url and plot(s) for displaying neural responses on the webpage
     capture error in each step and then log it to a dictionary (app_error_dict) in output
     """
 
+    @classmethod
+    def get_stim_type_list(cls):
+        """
+        get the list of stim_types defined by wormfunconn package
+        """
+        stim_type_list = ["rectangular", "delta", "sinusoidal", "realistic"]
+        return stim_type_list
```

Comment:
NB. Nice idea to encapsulate these into some functions, it seems to clean up the templates, etc. quite a bit. Would it make sense to make these part of the form class? On first glance, it seems to me like that would keep the form info all together in that class.

Reply:
I thought about it last Friday and decided to leave it in utils.py. In addition to forms.py,  this method is also called by "get_form_opt_field_dict" method in the same class in utils.py.